### ISSUE TITLE
v0.3.2 preg_match error

### ISSUE THREAD

**Issue by percymamedy** [GeneaLabs/laravel-caffeine#19]
I am getting this error with the v0.3.2

```php
ErrorException in LaravelCaffeineDripMiddleware.php line 32:
preg_match(): Delimiter must not be alphanumeric or backslash
```

v0.3.1 works fine

**Comment by IronSinew** [comment]
I'm also having this issue with 0.3.2. Rolled back to 0.3.1 for now.

**Comment by mikebronner** [comment]
Sorry about the delay on this. Been swamped. Will fix this ASAP. For some reason this was not an issue in my testing.

**Comment by mikebronner** [comment]
Just released v0.3.3 to address this. Thanks for your patience and reporting the problem :)
